### DISCUSSION TITLE
Second Pass options not showing up.

### DISCUSSION THREAD

**Discussion by BWDrum** [vladmandic/sdnext#1736] [Q&A]
About 75% of the time, after I launch the UI, the options under Second Pass do not pop-up. I feel like I might be missing a setting, but don't see anything. I'm using Firefox with all pop-up blockers disabled. TIA

**Comment by vladmandic** [comment]
options are closed by default and you need to click second pass to show them as its onclick event. if second pass is already enabled before you start ui, then you end up in your scenario. known limitation with any dynamically visible elements with gradio at the moment. will be addressed, but not near the list of priorities at the moment.

**Reply by BWDrum** [reply]
Thank you.

So, if I understand correctly, before I close the UI I should unclick Second Pass to ensure it functions the next time I launch?

**Reply by vladmandic** [reply]
or next time when you load ui, just uncheck and check it again.

**Reply by BWDrum** [reply]
Sadly, this doesn't work for me. Checking and uncheckinig does nothing and I can not access the second pass settings. BTW - Thank you for all of your hard work and support.

**Reply by BWDrum** [reply]
This issue seems to have been resolved. Second pass options are now showing reliably once again.
